spec/interactors/end_game/prepare_params_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+describe EndGame::PrepareParams do
+  include_context "with interactor"
+
+  let(:initial_context) do
+    {
+      current_user: current_user,
+      game_id: game_id
+    }
+  end
+  let(:current_user) { create :user }
+  let(:game_id) { 23 }
```

Comment:
```suggestion
```suggestion
  let(:game_id) { 23 }
```
```